kodein-di/src/commonMain/kotlin/org/kodein/di/bindProvider.kt

```diff
@@ -35,6 +35,8 @@ public inline fun <reified T: Any> DI.Builder.bindProvider(
     noinline creator: DirectDI.() -> T
 ): Unit = Bind(tag = tag, overrides = overrides, binding = provider(creator = creator))
 
+// region bindProviderOf overloads
+
 // TODO This fails with Kotlin/JS Legacy
 //  Re-enable this with IR only target.
```

Comment:
I'm not sure what this was referring to, but I removed the `noinline` call as it actually was not needed and was affecting performance negatively

Reply:
Sounds like it is an irrelevant comment, probably forgot to remove it.